pkg/api/v1/types.go

```diff
@@ -2131,6 +2131,11 @@ type PodAffinityTerm struct {
 	// for affinity and for RequiredDuringScheduling pod anti-affinity, empty topologyKey is not allowed.
 	// +optional
 	TopologyKey string `json:"topologyKey,omitempty" protobuf:"bytes,3,opt,name=topologyKey"`
+	// MinMatchingPods specifies the minimal number of pod that matching the selector; the term is considered to
```

Comment:
It's fine if you only handle anti-affinity case (that case is more important than affinity case). But make the comment clear (in the API and the code).

Or if you want to do both affinity and anti-affinity, that's fine too, but same comment applies. :)